src/demo/app/components/directive-button-demo.component.html

```diff
@@ -20,4 +20,12 @@ <h4><em>How it works</em></h4>
 <br>
 <pre style="margin-bottom: 0;">
     <code class="highcodeblock" [highlight]="code_bit_1"></code>
+</pre>
```

Comment:
don't like the way this looks, and the code wasn't what we wanted to show.  Here is what we want.

![image](https://user-images.githubusercontent.com/19155541/60884817-efa69800-a21b-11e9-88ef-4d67d5adfd8e.png)

and you can get the code part with this:
        this.code_bit_1 = `
      <button [keyboardShortcut]="keyboardShortcutDef" (click)="alertMessage()">
          Show the Message
      </button>
      `;
        this.code_bit_2 = `
      keyboardShortcutDef: IKeyboardShortcutListenerOptions = {
        description: 'simple, individual, demo shortcut',
        keyBinding: [KeyboardKeys.Ctrl, 'i'],
      };
      `;

Reply:
Just updated this should look the way you want it to now.